libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFEncryptionKey.h

```diff
@@ -49,14 +49,14 @@ NS_ASSUME_NONNULL_BEGIN
  @param dataToEncrypt The data to encrypt.
  @returns `NSData` object that contains the encrypted data.
  */
-- (NSData*)encryptData:(NSData *)dataToEncrypt;
+- (nullable NSData *)encryptData:(NSData *)dataToEncrypt;
```

Comment:
The methods that these call into are nullable but if it's too much of a semver issue I can leave them